provision/dev/values/llmariner-worker-plane.yaml

```diff
@@ -18,6 +18,8 @@ inference-manager-engine:
 job-manager-dispatcher:
   notebook:
     llmarinerBaseUrl: http://control-plane/v1
+  # TODO(kenji): Consider changing the default values in the chart.
```

Comment:
@Ladicle , mostly fyi, but we will need to make the update happen frequently in the demo  setting.